amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -3393,8 +3393,8 @@ type StreamingPayment @model {
   createdAt: AWSDateTime!
   recipientAddress: String!
   nativeDomainId: Int!
-  startTime: AWSTimestamp!
-  endTime: AWSTimestamp!
+  startTime: String!
+  endTime: String!
```

Comment:
What's the reason we're using plain strings to store the timestamps as opposed to the native AWS type ?

If it's because we're storing timestamps rather than actual time, maybe we should rename them to reflect that fact.

Reply:
So the AWSTimestamp scalar wouldn't allow anything bigger than a 32 bit number, and since we're now using potentially 256 bits I had to change to a string.